src/components/Logo/index.tsx

```diff
@@ -24,11 +28,12 @@ const Img = styled.img`
 
 interface LogoProps {
   isBackgroundDark: boolean;
+  width?: string;
```

Comment:
Normally I always put interfaces at the beginning of the file, this way you already know what are the needs of given element. We can agree to some convention, maybe we'd like to extract interfaces to separate files/keep them grouped at the top or leave it this way? @evulins @bazelajoanna

Reply:
I am ok with both - moving to the top and also moving to a different file.

Reply:
FYI Agreed to extract it.